src/utils/notify.rs

```diff
@@ -1,8 +1,34 @@
+use std::fmt;
+
 #[derive(Debug)]
 pub(crate) enum NotificationLevel {
+    Debug,
     Verbose,
     Info,
     Warn,
     Error,
-    Debug,
+}
+
+impl fmt::Display for NotificationLevel {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str(match self {
+            NotificationLevel::Debug => "debug",
+            NotificationLevel::Verbose => "verbose",
+            NotificationLevel::Info => "info",
+            NotificationLevel::Warn => "warning",
+            NotificationLevel::Error => "error",
+        })
+    }
+}
+
+impl From<tracing::Level> for NotificationLevel {
+    fn from(level: tracing::Level) -> Self {
+        match level {
+            tracing::Level::TRACE => Self::Debug,
```

Comment:
I'd probably import `tracing::Level` in this module.

Reply:
Hmmm, I guess the rationale here is that `Level` is ambiguous since there are essentially two level types and if I directly `use tracing::Level` it seems less clear... I've considered `use tracing::Level as TracingLevel` but this is not doing any good.